examples/cookbook/effects/expandable_fab/lib/excerpt1.dart

```diff
@@ -15,10 +15,10 @@ class ExpandableFab extends StatefulWidget {
   final List<Widget> children;
 
   @override
-  _ExpandableFabState createState() => _ExpandableFabState();
+  ExpandableFabState createState() => ExpandableFabState();
```

Comment:
The correct fix for this line is instead:
```dart
State<ExpandableFab> createState() => _ExpandableFabState();
```

Reply:
I made the changes you suggested to all Dart files and manually to the MD file. Is there any reading you could point me to on understanding this solution over my original attempt?

Reply:
Books on software API design have been few and far between for the last couple of decades. It's almost as if everyone has gone nuts with network APIs or something. I'd look to some of the classics like Code Complete and Beautiful Code for ideas in this space.